### PR TITLE
Updating logic for opening ECS logs so that we fall-back to log group

### PR DESCRIPTION
**Pull request by kiiadi**
Open a single-stream if it exists, else fall-back to the log-group.

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 1]
```diff
@@ -44,7 +44,7 @@ lambda.function.iamRole.tooltip=The function's execution role.
 lambda.function.enableXRay.tooltip=Enables AWS X-Ray to detect, analyze, and optimize performance issues when this function runs.
 lambda.function.sourceBucket.tooltip=The Amazon S3 bucket in the same AWS Region to deploy this AWS Lambda function.
 lambda.create_new=Create new AWS Lambda...
-lambda.logs.does_not_exist=Lambda has not sent events to CloudWatch
+lambda.logs.does_not_exist=Lambda '''{0}''' has not sent events to CloudWatch
```
Comment (hunterwerlla) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties:47]: What does the triple single quotes do?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt [hunk 2]
```diff
@@ -83,18 +84,28 @@ class ContainerLogsAction(
...
+
+                }
+                window.showLogGroup(logGroup)
             }
-            window.showLogStream(logGroup, logStream)
+    }
+
+    private fun showSingleStream(window: CloudWatchLogWindow, logGroup: String, logStream: String): Boolean {
+        if (!project.awsClient<CloudWatchLogsClient>().checkIfLogStreamExists(logGroup, logStream)) {
+            notifyInfo(message("ecs.service.logs.no_log_stream"))
```
Comment (hunterwerlla) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt:105]: We should update the message to say "showing log streams for the cluster instead" or similar

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt [hunk 3]
```diff
@@ -83,18 +84,28 @@ class ContainerLogsAction(
...
-            }
-            val logStream = "$logPrefix/${container.containerDefinition.name()}/$taskId"
-            if (!project.awsClient<CloudWatchLogsClient>().checkIfLogStreamExists(logGroup, logStream)) {
-                notifyError(message("ecs.service.logs.no_log_stream"))
-                return@thenAccept
+        AwsResourceCache.getInstance(project)
+            .getResource(EcsResources.listTaskIds(container.service.clusterArn(), container.service.serviceArn()))
+            .thenAccept { tasks ->
+                when {
+                    tasks.isEmpty() -> notifyInfo(message("ecs.service.logs.no_running_tasks"))
```
Comment (hunterwerlla) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/ContainerActions.kt:91]: We should update the message to say "no running tasks, showing log streams for cluster" or similar

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties (hunk 1 above)
Comment (kiiadi) [reply resources/resources/software/aws/toolkits/resources/localized_messages.properties:47]: oops should be double quotes

**Pull request merged**